io.github.narunlifescience.AlphaPlot.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be upstreamed if possible.

Reply:
Just for the record, I'm not the upstream of this project, but he knows about this PR and [agreed to package AlphaPlot in flatpak](https://github.com/narunlifescience/AlphaPlot/issues/20#issue).

I'm going to talk to the upstream to see if it accepts adding the appdata in the source code.

Reply:
It was upstreamed ([commit](https://github.com/narunlifescience/AlphaPlot/commit/70c42625fbe3a3ca81875b27d1ca93efb2955d9d))